### PR TITLE
Modify add command to handle floating tasks

### PR DESCRIPTION
**Pull request by lamgj**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
-            + "Parameters: NAME p/PHONE e/EMAIL a/ADDRESS  [t/TAG]...\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to the task manager. "
+            + "Parameters: NAME p/PRIORITY s/STATUS n/NOTES  [t/TAG]...\n"
             + "Example: " + COMMAND_WORD
-            + " John Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney";
+            + " Finish assignment p/HIGH s/INCOMPLETE n/Due 30/02/2017 t/CS1234";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
+    public static final String MESSAGE_SUCCESS = "New task added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
```
Comment (mynameisyz) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: Remember to change old references of person and address book to task/task manager in line 30

Comment (lamgj) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: ok noted!

File: src/main/java/seedu/address/model/task/Priority.java [hunk 2]
```diff
@@ -8,8 +8,8 @@
  */
 public class Priority {
 
-    public static final String MESSAGE_PRIORITY_CONSTRAINTS = "Task priority should only contain numbers";
-    public static final String PRIORITY_VALIDATION_REGEX = "\\d+";
+    public static final String MESSAGE_PRIORITY_CONSTRAINTS = "Task priority can take any values";
```
Comment (xbili) [inline src/main/java/seedu/address/model/task/Priority.java:11]: This one is tricky, but should we allow users to input any value or a numeric value from 1 to 5?

Comment (xbili) [reply src/main/java/seedu/address/model/task/Priority.java:11]: Okay sorry, I just remembered that we are allowing all values for `v0.1`. Will add an issue for this for `v0.2`.

**Review by xbili — APPROVED** [review]
Looks good to me!

**Pull request merged**